tests/config/config_test.yaml

```diff
@@ -14,6 +14,7 @@ services:
         autowire: true
 
     Symfony\Component\Security\Core\Security: null
```

Comment:
Shouldn't this line be removed?

Reply:
Right!. Security service is not needed anymore. Fixed.